docs/_docs/continuous-integration.md

```diff
@@ -1,232 +1,20 @@
-access to Bundler, RubyGems, and a Ruby runtime.
-
-```yaml
-rvm:
-- 2.2.5
-```
-
-RVM is a popular Ruby Version Manager (like rbenv, chruby, etc). This
-directive tells Travis the Ruby version to use when running your test
-script.
-
-```yaml
-before_script:
- - chmod +x ./script/cibuild
-```
-
-The build script file needs to have the *executable* attribute set or
-Travis will fail with a permission denied error. You can also run this
-locally and commit the permissions directly, thus rendering this step
-irrelevant.
-
-```yaml
-script: ./script/cibuild
-```
-
-Travis allows you to run any arbitrary shell script to test your site. One
-convention is to put all scripts for your project in the `script`
-directory, and to call your test script `cibuild`. This line is completely
-customizable. If your script won't change much, you can write your test
-incantation here directly:
-
-```yaml
-install: gem install jekyll html-proofer
-script: jekyll build && htmlproofer ./_site
-```
-
-The `script` directive can be absolutely any valid shell command.
-
-```yaml
-# branch whitelist, only for GitHub Pages
-branches:
-  only:
-  - gh-pages     # test the gh-pages branch
-  - /pages-(.*)/ # test every branch which starts with "pages-"
-```
-
-You want to ensure the Travis builds for your site are being run only on
-the branch or branches which contain your site. One means of ensuring this
-isolation is including a branch whitelist in your Travis configuration
-file. By specifying the `gh-pages` branch, you will ensure the associated
-test script (discussed above) is only executed on site branches. If you use
-a pull request flow for proposing changes, you may wish to enforce a
-convention for your builds such that all branches containing edits are
-prefixed, exemplified above with the `/pages-(.*)/` regular expression.
-
-The `branches` directive is completely optional. Travis will build from every
-push to any branch of your repo if leave it out.
-
-```yaml
-env:
-  global:
-  - NOKOGIRI_USE_SYSTEM_LIBRARIES=true # speeds up installation of html-proofer
-```
-
-Using `html-proofer`? You'll want this environment variable. Nokogiri, used
-to parse HTML files in your compiled site, comes bundled with libraries
-which it must compile each time it is installed. Luckily, you can
-dramatically decrease the install time of Nokogiri by setting the
-environment variable `NOKOGIRI_USE_SYSTEM_LIBRARIES` to `true`.
-
-<div class="note warning">
-  <h5>Be sure to exclude <code>vendor</code> from your
-   <code>_config.yml</code></h5>
-  <p>Travis bundles all gems in the <code>vendor</code> directory on its build
-   servers, which Jekyll will mistakenly read and explode on.</p>
+Continuous Integration (CI), and related concepts, enables you to publish your Jekyll generated website with confidence by automating the quality assurance (QA) and deployment processes. You can quickly get started using CI with one of the providers below:
```

Comment:
Ah okay. My thought was to just teach what the abbreviation was but this makes sense.